src/sparse_blas/backends/mklcpu/mklcpu_operations.cpp

```diff
@@ -0,0 +1,209 @@
+    sycl::buffer<fpType, 1>& /*y*/) {
+    throw unimplemented("SPARSE_BLAS", "gemv");
+}
+
+template <typename fpType>
+std::enable_if_t<detail::is_fp_supported_v<fpType>, sycl::event> gemv(
+    sycl::queue& /*queue*/, transpose /*transpose_val*/, const fpType /*alpha*/,
+    detail::matrix_handle* /*A_handle*/, const fpType* /*x*/, const fpType /*beta*/,
+    const fpType* /*y*/, const std::vector<sycl::event>& /*dependencies*/) {
+    throw unimplemented("SPARSE_BLAS", "gemv");
+}
+
+template <typename fpType>
+std::enable_if_t<detail::is_fp_supported_v<fpType>> gemvdot(
+    sycl::queue& /*queue*/, transpose /*transpose_val*/, fpType /*alpha*/,
+    detail::matrix_handle* /*A_handle*/, sycl::buffer<fpType, 1>& /*x*/, fpType /*beta*/,
+    sycl::buffer<fpType, 1>& /*y*/, sycl::buffer<fpType, 1>& /*d*/) {
+    throw unimplemented("SPARSE_BLAS", "gemvdot");
+}
+
+template <typename fpType>
+std::enable_if_t<detail::is_fp_supported_v<fpType>, sycl::event> gemvdot(
+    sycl::queue& /*queue*/, transpose /*transpose_val*/, fpType /*alpha*/,
+    detail::matrix_handle* /*A_handle*/, fpType* /*x*/, fpType /*beta*/, fpType* /*y*/,
+    fpType* /*d*/, const std::vector<sycl::event>& /*dependencies*/) {
+    throw unimplemented("SPARSE_BLAS", "gemvdot");
+}
+
+template <typename fpType>
+std::enable_if_t<detail::is_fp_supported_v<fpType>> symv(sycl::queue& /*queue*/, uplo /*uplo_val*/,
+                                                         fpType /*alpha*/,
+                                                         detail::matrix_handle* /*A_handle*/,
+                                                         sycl::buffer<fpType, 1>& /*x*/,
+                                                         fpType /*beta*/,
+                                                         sycl::buffer<fpType, 1>& /*y*/) {
+    throw unimplemented("SPARSE_BLAS", "symv");
+}
+
+template <typename fpType>
+std::enable_if_t<detail::is_fp_supported_v<fpType>, sycl::event> symv(
+    sycl::queue& /*queue*/, uplo /*uplo_val*/, fpType /*alpha*/,
+    detail::matrix_handle* /*A_handle*/, fpType* /*x*/, fpType /*beta*/, fpType* /*y*/,
+    const std::vector<sycl::event>& /*dependencies*/) {
+    throw unimplemented("SPARSE_BLAS", "symv");
+}
+
+template <typename fpType>
+std::enable_if_t<detail::is_fp_supported_v<fpType>> trmv(
+    sycl::queue& /*queue*/, uplo /*uplo_val*/, transpose /*transpose_val*/, diag /*diag_val*/,
+    fpType /*alpha*/, detail::matrix_handle* /*A_handle*/, sycl::buffer<fpType, 1>& /*x*/,
+    fpType /*beta*/, sycl::buffer<fpType, 1>& /*y*/) {
+    throw unimplemented("SPARSE_BLAS", "trmv");
+}
+
+template <typename fpType>
+std::enable_if_t<detail::is_fp_supported_v<fpType>, sycl::event> trmv(
+    sycl::queue& /*queue*/, uplo /*uplo_val*/, transpose /*transpose_val*/, diag /*diag_val*/,
+    fpType /*alpha*/, detail::matrix_handle* /*A_handle*/, fpType* /*x*/, fpType /*beta*/,
+    fpType* /*y*/, const std::vector<sycl::event>& /*dependencies*/) {
```

Comment:
Did you manually add the `/*` and `*/`? There's a whole lot of that you might have had to do! Do you happen to be using a script or something to do this?